src/app/People/people.component.scss

```diff
@@ -0,0 +1,79 @@
+.page-header{
+    padding: 1em;
+    clear: left;
+    text-align: center;
+    padding-right: 3%;
+    font-size: 20px;
+    box-shadow: 0px 1px 5px black;
+    width: 1120px;
```

Comment:
Use responsive CSS.
Avoid hard coded values, its not going to work in other displays